src/containers/home/home.component.jsx

```diff
@@ -1,25 +1,42 @@
-import React from "react";
-import { useSelector } from "react-redux";
+import React, { useEffect, useState } from "react";
 
-import SampleDummy from "components/sampleDummy";
+import { signInWithGoogle, auth } from "database";
 
-const Home = (props) => {
-  const { fetchUserThunk } = props;
+const Home = () => {
+  const [user, setUser] = useState({});
 
-  // This is how to fetch data from the global state
-  const user = useSelector((state) => state.users.user);
+  useEffect(() => {
+    auth.onAuthStateChanged(async (nextUser) => {
```

Comment:
This function will run every time there is a change in the auth object (e.g. When a user is signed in, signed out, etc.)